src/Components/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useContext, useMemo, useState } from 'react';
+import { TodoList } from '../TodoList/TodoList';
+import { TodoFilter } from '../TodoFilter/TodoFilter';
+import { TodosContext } from '../../Store';
+
+export const TodoApp: React.FC = () => {
+  // eslint-disable-next-line prettier/prettier
+  const { addTodo, todos, removeCompletedTodos, remainingTodos, toggleAll } =
+    useContext(TodosContext);
```

Comment:
still not fixed 
![image](https://github.com/mate-academy/react_todo-app/assets/70009205/25fd1e10-d3f6-4a5b-91e7-697b69b6aafc)